src/components/Tile/Tile-story.js

```diff
@@ -8,6 +9,8 @@ import {
   TileAboveTheFoldContent,
   TileBelowTheFoldContent,
 } from '../Tile';
+import TileGroup from '../TileGroup';
+import TileRadio from '../TileRadio';
```

Comment:
To keep with the current convention, could we name this `RadioTile`? I agree, not the greatest naming convention, but will have to do. Can we keep track of this somewhere to make this change in v9 @marijohannessen ?

Reply:
We have a `v9` label that we can throw on issues, so once we decide on naming here we can create one